modules/Bio/Vega/Transform/RegionToXML.pm

```diff
@@ -209,6 +203,21 @@ sub generate_Locus {
         $g->indent($indent);
     }
 
+    # Fetch gene attributes that do not persist.
+    my @current_persistent_attributes = ['name', 'remark', 'hidden_remark', 'cds_start_NF', 'cds_end_NF', 'mRNA_start_NF', 'mRNA_end_NF', 'status', 'synonym', 'otter_truncated'];
```

Comment:
It seems to me that you are storing a reference to an array into an array, I don't see any reason/advantage